src/wp-includes/wp-db.php

```diff
@@ -3861,12 +3854,14 @@ public function get_charset_collate() {
 	 * @since 2.7.0
 	 * @since 4.1.0 Added support for the 'utf8mb4' feature.
 	 * @since 4.6.0 Added support for the 'utf8mb4_520' feature.
+	 * @since 6.0.0 Added support for the 'identifier_placeholders' feature.
```

Comment:
I think the description of this dockblock needs an update.

Replace `* Determines if a database supports a particular feature.` 

With
```
* Determine DB or WPDB support for a particular feature.
*
* Capability sniffs for the database server and current version of wpdb. 
* 
* Database sniffs test based on the version of MySQL the site is using.
* 
* WPDB sniffs are added as new features are introduced to allow theme and plugin
* developers to determine feature support. This is to account for drop-ins which may
* introduce feature support at a different time to WordPress.

Reply:
Good idea, has been updated.